<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="headerLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Header</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="headerCharacterLayout">
       <item>
        <widget class="QLabel" name="headerCharacterLabel">
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Header Character</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="headerCharacterEdit">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="headerTemplateLayout">
       <item>
        <widget class="QLabel" name="headerTemplateLabel">
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Header Template</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="headerTemplateEdit">
         <property name="text">
          <string>seq</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="fixedSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="resultsLayout">
     <item>
      <widget class="QLabel" name="resultsLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Results</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="prefixLayout">
       <item>
        <widget class="QLabel" name="prefixLabel">
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Prefix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="prefixEdit">
         <property name="text">
          <string>Result_</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="formatLayout">
       <item>
        <widget class="QLabel" name="formatLabel">
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Fomat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="formatEdit">
         <property name="text">
          <string>fas</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="directoryLayout">
       <item>
        <widget class="QLabel" name="directoryLabel">
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="directoryEdit">
         <property name="text">
          <string>./results/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
